CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ## Unreleased
 
 * Step by step nav header updated to use heading ([PR #1671](https://github.com/alphagov/govuk_publishing_components/pull/1671))
+* Increase icon size and allow transparent icons on action link component ([PR #1674](https://github.com/alphagov/govuk_publishing_components/pull/1671))
```

Comment:
This should match the PR title (or vice versa - this seems like a good description).